drivers/lhal/include/bflb_adc.h

```diff
@@ -40,6 +40,7 @@
 /** @defgroup ADC_CLK_DIV adc clock divison definition
   * @{
   */
+#define ADC_CLK_DIV_1  0
```

Comment:
give up div1

Reply:
Pleas explain. This is valid divider value for ADC clock and I'm using it in my project.

Reply:
adc value is not inaccuracy in div1 in some cases.

Reply:
Yes, of course. But ADC speed should match external signal driver impedance to get proper signal sampling. So, it is up to board and software developer to select proper divider value for their signal parameters to get accurate ADC sampling and reading. So I suppose, that we should not limit this parameter in SDK if it is allowed in datasheet and reference manual and there is no restrictions in errata.

Reply:
For div1, we will remove in datasheet.